docs/sources/writing-guide/front-matter/index.md

```diff
@@ -34,54 +34,45 @@ Here’s a correctly built example:
-        </tr>
-        <tr>
-            <td align="left" valign="top">[aliases]</td>
-            <td align="left" valign="top">Provides an HTML redirect from the pages in the list to the current page.</td>
-            <td align="left" valign="top">Described in detail in <a href="#hugo-aliases">Hugo aliases</a>.
-            </td>
-        </tr>
-        <tr>
-            <td align="left" valign="top">[weight]</td>
-            <td align="left" valign="top">The [weight] determines the placement of the topic within the left-hand sidebar of our website, with smaller numbers placing the topic higher in the guide. <br><br>Pages with the same weight have lexicographic ordering. </td>
-            <td align="left" valign="top">Use increments of `100` for all other content files, because doing so eliminates much of the need to re-order existing topics when new topics are added. <br><br>Weights are per web directory.</a>
-            </td>
-        </tr>
-        <tr>
-            <td align="left" valign="top">[keywords]</td>
-            <td align="left" valign="top">Keywords are used by the website to link to related pages in the “related content” sections. https://github.com/grafana/website/blob/master/config/_default/config.yaml#L85 <br><br>They do not appear in the resulting HTML source for the page and have no effect on SEO.</td>
-            <td align="left" valign="top">Ideally, use single terms as opposed to phrases.</a>
-            </td>
-        </tr>
-    </tbody>
-</table>
-
-## [Title] v [MenuTitle] example
+The following list describes what each element does and provides guidelines for its content.
+
+`title` **Required.**
+
+:  Becomes the document title element. Often browsers display this in the tab for the page.
+
+   It doesn't need to precisely match the `menuTitle`.
+   The title should be optimized for search engines. May be surrounded by double quote marks ("). Do not use smart quotes.
+
+`description`
+
+:  Displays as a clue to users about what the page should include on social (Twitter and the like),
+   though not as much by a search engine.
+
+   The number of characters vary by media, but use them wisely.
+   Provide enough information to guide users to the content by describing what content is provided using the link.
+   Often, this doesn’t need to be original prose&mdash;you can often scan the first few paragraphs to pluck the appropriate terms or phrases into the description.
+   It won't cause harm if it's too long, it will simply truncate in the displayed media.
+   May be surrounded by double quote marks ("). Do not use smart quotes.
```

Comment:
```suggestion
   Use double quotes (`"`) to surround the title. Do not use smart quotes.
```
Writing a happy path. Less thinking if we say, "Use ..." instead of "You can use ..."